Kopernicus/Kopernicus/RuntimeUtility/RnDFixer.cs

```diff
@@ -40,6 +40,18 @@ namespace Kopernicus
     /// </summary>
     public class RnDFixer : MonoBehaviour
     {
+        List<RDPlanetListItemContainer> stars = new List<RDPlanetListItemContainer>();
+
+        void LateUpdate()
+        {
+            // Block the orientation of all stars
+            for (int i = 0; i < stars?.Count(); i++)
```

Comment:
`stars` is never null so you can remove the `?` (I am suprised that this compiles, since you are comparing an int to null)

Reply:
an int cannot be lower than a null so it returns false

I had an issue once where a null list was breaking my code, since then I've always used ```?``` to make sure that even if I have a null the .Count() method doesn't trigger a nullref

is there a downside to using the ```?``` ?

Reply:
It is extra code that gets executed every frame and that isn't neccessary.